src/community/index.md.njk

```diff
@@ -9,9 +9,10 @@ The GOV.UK Design System is for everyone. It has a strong cross-government commu
 
 Find out:
 
+- about useful [community resources](/community/resources)
```

Comment:
```suggestion
- about [community resources](/community/resources)
```

It's best to avoid adjectives - we're not the judge of whether they're useful or not, our users are :)